scripts/quests/bastok/A_Ladys_Heart.lua

```diff
@@ -92,8 +92,8 @@ quest.sections =
                     -- Correct trade option (Amaryllis).
                     if option == 2002 then
                         if quest:complete(player) then
-                            player:confirmTrade()
```

Comment:
Don't remove the confirmTrade, Sir. It's needed.

Reply:
Whoops, good catch 👍